### PR TITLE
Update gpdemo documentation

### PR DESCRIPTION
**Pull request by andreasscherbaum**
Remove Solaris documentation
Update port numbers
Add environment variables

### CODE REVIEW

File: gpAux/gpdemo/README [hunk 1]
```diff
@@ -33,33 +30,35 @@ RUNNING GP DEMO
...
 
-	tar -xvzf $GPHOME/gpMgmt/demo/gpdemo.tar.gz
+	. /usr/local/greenplum-db/greenplum_path.sh
 
-4. Go into the gpdemo directory you just untarred:
+   Note: There is a space between the dot and the slash.
 
-	cd gpdemo
+   The "." will source the greenplum_path.sh into the current
+   shell, instead of staring a subprocess.
```
Comment (danielgustafsson) [inline gpAux/gpdemo/README]: s/staring/starting/

File: gpAux/gpdemo/README [hunk 2]
```diff
@@ -2,28 +2,25 @@
...
 
-The GP demo program will setup a virtual Greenplum Database system on a 
-single host. It creates and starts a GP master instance and three 
-segment instances within the same directory. Running GP demo is a good 
-way to test your initial installation before deploying the Greenplum Database 
-software on a multi-host array. It is also a good way to become familiar 
-with the system without investing the time involved in a multi-host setup.
+The GP demo program will setup a virtual Greenplum Database system
+on a single host. It creates and starts a GP master instance and
+three segment instances within the same directory. Running GP demo
```
Comment (danielgustafsson) [inline gpAux/gpdemo/README]: Should it be mentioned that it's three segments with mirrors?

File: gpAux/gpdemo/README (hunk 1 above)
Comment (cjcjameson) [inline gpAux/gpdemo/README]: got it in my typo commit

**Comment by andreasscherbaum** [comment]
Incorporated changes from @cjcjameson.
Anyone else wants to review, else I merge this in the next few days.

File: gpAux/gpdemo/README (hunk 2 above)
Comment (andreasscherbaum) [inline gpAux/gpdemo/README]: Done

**Pull request merged**